src/main/java/org/LogAggregatorTool/LogAggregatorTool.java

```diff
@@ -0,0 +1,41 @@
+package org.LogAggregatorTool;
+
+import org.LogAggregatorTool.operations.LogFilesExecutor;
+import org.LogAggregatorTool.utility.Constants;
+import java.io.File;
+import java.util.Scanner;
+
+public class LogAggregatorTool {
+
+    public static void main(String[] args){
```

Comment:
Keep the main() clean. Multiple functionalities inside main(0 is not recommended